pkg/standalone/run.go

```diff
@@ -79,12 +79,16 @@ func (meta *DaprMeta) newAppID() string {
 	}
 }
 
-func (config *RunConfig) validateComponentPath() error {
-	_, err := os.Stat(config.ComponentsPath)
+func (config *RunConfig) validateResourcesPath() error {
+	dirPath := config.ResourcesPath
+	if dirPath == "" {
+		dirPath = config.ComponentsPath
+	}
+	_, err := os.Stat(dirPath)
```

Comment:
This is the function that is introducing the E2E change seen in this PR. Previously resources path value was not validated. Now it is being validated.